Library/Homebrew/formula.rb

```diff
@@ -3360,24 +3339,6 @@ def patch(strip = :p1, src = nil, &block)
       specs.each { |spec| spec.patch(strip, src, &block) }
     end
 
-    # Defines launchd plist handling.
-    #
-    # Does your plist need to be loaded at startup?
-    # <pre>plist_options startup: true</pre>
-    #
-    # Or only when necessary or desired by the user?
-    # <pre>plist_options manual: "foo"</pre>
-    #
-    # Or perhaps you'd like to give the user a choice? Ooh fancy.
-    # <pre>plist_options startup: true, manual: "foo start"</pre>
-    #
-    # @deprecated Please use {Homebrew::Service.require_root} instead.
-    def plist_options(options)
-      odisabled "plist_options", "service.require_root"
```

Comment:
Given the number of errors in audit CI here: it seems this deprection didn't work and we will probably have to restart the process and figure out why.

Reply:
Ah right it's because of; https://github.com/Homebrew/brew/blob/c32bd1c7cc0c3b1b914845bddfeda53f4d877a3f/Library/Homebrew/extend/kernel.rb#L129

Which doesn't really worth with things that are future-deprecated.

Reply:
Handled the remaining formulae in https://github.com/Homebrew/homebrew-core/pull/156780.